beanstalk_worker/worker_start_build.py

```diff
@@ -128,9 +128,15 @@ def start_build(job_details):
 while True:
     try:
         debug_log("listening to start_build tube")
-        job = bs.reserve()
-        job_details = json.loads(job.body)
-        result = start_build(job_details)
-        job.delete()
+        current_jobs_in_tube = int(dict(item.split(":") for item in bs.stats_tube('start_build').split('\n')[1:-1])['current-jobs-ready'])
+        if current_jobs_in_tube > 0 :
+            job = bs.reserve()
+            job_details = json.loads(job.body)
+            result = start_build(job_details)
+            job.delete()
+        else:
+            debug_log("No job found to process looping again")
+            time.sleep(30)
     except Exception as e:
         logger.log(level=logging.CRITICAL, msg=e.message)
+        time.sleep(30)
```

Comment:
There are multiple occurrences of `time.sleep(30)` in the code. Can we use a variable instead, so that, if in future we decide to change the value, we just need to change it at one place instead of remembering to change at muliple places?